website/scripts.js

```diff
@@ -0,0 +1,69 @@
+import { execSync } from "child_process";
+
+class Scripts {
```

Comment:
Can you add inline comments to provide more context for users who might not be familiar with plugins. A few things to cover...
- Why it exists - to run custom scripts, which we need to run the build script at deploy time, with support for variable resolution from framework, as well as proper stdio handling.
- How this works - it runs because it's included as a plugin in serverless.yml, and how hooks work.